pkg/clustermanager/e2e-tests/get.go

```diff
@@ -14,16 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package clustermanager
+package e2e_tests
 
 import (
-	clm "knative.dev/test-infra/pkg/clustermanager/gke"
+	clm "knative.dev/test-infra/pkg/clustermanager/e2e-tests/gke"
 )
 
 // Get gets a GKE cluster
-func (rw *RequestWrapper) Get() (*clm.GKECluster, error) {
+func Get(rw *RequestWrapper) (*clm.GKECluster, error) {
 	rw.Request.SkipCreation = true
 	// Reuse `Create` for getting operation, so that we can reuse the same logic
 	// such as protected project/cluster etc.
-	return rw.Create()
+	return Create(rw)
```

Comment:
Nit: I'm not sure why you're moving from a method to a function-based approach. They are, in essence, identical but this might introduce aliasing issues in the future with the function names. Could you provide some insight on your decision?

Reply:
The reason I'm making this change is one of our other repos are importing these functions and having some unit tests around them, having them as methods won't be unit-testable.
And different packages can have functions with the same name, so function names shouldn't be a problem.